### ISSUE TITLE
Most markers collections lack READMEs

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/datastore-issues#74]
I cannot load collections into mines any longer if they lack a README, so the following collections need READMEs. I can handle it, but I'll let @cann0010 have first dibs since he knows much more about them than I do. The only important extra thing is to have in the README is the **genotyping_platform** so we can match up genetic data that reference the marker set, e.g. `genotyping_platform: SoySNP50K`. Otherwise a bit of research on the marker set is required if we want to list a publication, etc. The required README attributes must be in there, of course. (I'll add a check to the mine GFF loader to be sure the genotyping_platform is present.)
```
Arachis/hypogaea/markers/
├── Tifrunner.gnm1.mrk.Axiom_Arachis_58K_SNP
│   └── arahy.Tifrunner.gnm1.mrk.Axiom_Arachis_58K_SNP.gff3.gz
├── Tifrunner.gnm1.mrk.Axiom_Arachis2_SNP
│   └── arahy.Tifrunner.gnm1.mrk.Axiom_Arachis2_SNP.gff3.gz
└── Tifrunner.gnm1.mrk.TPop_map_Agarwai2018
    └── arahy.Tifrunner.gnm1.mrk.TPop_map_Agarwai2018.gff3.gz
Glycine/max/markers/
├── Wm82.gnm1.mrk.SoySNP50K
│   └── glyma.Wm82.gnm1.mrk.GHRN.SoySNP50K.gff3.gz
├── Wm82.gnm1.mrk.SoySNP6K
│   └── glyma.Wm82.gnm1.mrk.GHRN.SoySNP6K.gff3.gz
├── Wm82.gnm2.mrk.1536_USLP1
│   └── glyma.Wm82.gnm2.mrk.1536_USLP1.gff3.gz
├── Wm82.gnm2.mrk.Li_Zhao_2019
│   └── glyma.Wm82.gnm2.mrk.Li_Zhao_2019.gff3.gz
├── Wm82.gnm2.mrk.NJAU355K
│   └── glyma.Wm82.gnm2.mrk.XJP2.NJAU355K.gff3.gz
├── Wm82.gnm2.mrk.SLAFseq
│   └── glyma.Wm82.gnm2.mrk.SLAFseq.gff3.gz
├── Wm82.gnm2.mrk.SoyaSNP180K
│   └── glyma.Wm82.gnm2.mrk.SoySNP50K.gff3.gz
├── Wm82.gnm2.mrk.SoySNP50K
│   └── glyma.Wm82.gnm2.mrk.SoyaSNP180K.gff3.gz
├── Wm82.gnm2.mrk.SoySNP6K
│   └── glyma.Wm82.gnm2.mrk.SoySNP6K.gff3.gz
└── Wm82.gnm2.mrk.various
    └── glyma.Wm82.gnm2.mrk.various.gff3.gz
Phaseolus/vulgaris/markers/
├── G19833.gnm1.mrk.BARCBean6K
│   └── phavu.G19833.gnm1.mrk.BARCBean6K.gff3.gz
├── G19833.gnm1.mrk.Benchimol_deCampos_2007
│   └── G19833.gnm1.mrk.Benchimol_deCampos_2007.gff3.gz
├── G19833.gnm1.mrk.Blair_Pedraza_2003
│   └── G19833.gnm1.mrk.Blair_Pedraza_2003.gff3.gz
├── G19833.gnm1.mrk.Cardoso_Oblessuc_2008
│   └── phavu.G19833.gnm1.mrk.Cardoso_Oblessuc_2008.gff3.gz
├── G19833.gnm1.mrk.Gaitan-Solis_Duque_2002
│   └── phavu.G19833.gnm1.mrk.Gaitan-Solis_Duque_2002.gff3.gz
├── G19833.gnm1.mrk.Galeano_Cortes_2012
│   └── G19833.gnm1.mrk.Galeano_Cortes_2012.gff3.gz
├── G19833.gnm1.mrk.Galeano_Fernandez_2011
│   └── phavu.G19833.gnm1.mrk.Galeano_Fernandez_2011.gff3.gz
├── G19833.gnm1.mrk.Grisi_Blair_2007
│   └── G19833.gnm1.mrk.Grisi_Blair_2007.gff3.gz
├── G19833.gnm1.mrk.Hanai_deCampos_2007
│   └── G19833.gnm1.mrk.Hanai_deCampos_2007.gff3.gz
├── G19833.gnm1.mrk.Hougaard_Madsen_2008
│   └── G19833.gnm1.mrk.Hougaard_Madsen_2008.gff3.gz
├── G19833.gnm1.mrk.Kim_Kaur_2006
│   └── G19833.gnm1.mrk.Kim_Kaur_2006.gff3.gz
├── G19833.gnm1.mrk.McClean_Mamidi_2010
│   └── G19833.gnm1.mrk.McClean_Mamidi_2010.gff3.gz
├── G19833.gnm1.mrk.Murray_Larsen_2002
│   └── G19833.gnm1.mrk.Murray_Larsen_2002.gff3.gz
├── G19833.gnm1.mrk.Parker_BernyMieryTeran_2020
│   └── phavu.G19833.gnm1.mrk.Parker_BernyMieryTeran_2020.gff3.gz
├── G19833.gnm1.mrk.Parker_BernyMierYTeran_2020
│   └── phavu.G19833.gnm1.mrk.Parker_BernyMierYTeran_2020.gff3.gz
├── G19833.gnm1.mrk.PvCookUCDavis2009
│   └── phavu.G19833.gnm1.mrk.PvCookUCDavis2009.gff3.gz
├── G19833.gnm1.mrk.Raggi_Caproni_2019
│   └── phavu.G19833.gnm1.mrk.Raggi_Caproni_2019.gff3.gz
├── G19833.gnm1.mrk.Yu_Park_2000
│   └── G19833.gnm1.mrk.Yu_Park_2000.gff3.gz
├── G19833.gnm2.mrk.DiVittori_Bitocchi_2021
│   └── phavu.G19833.gnm2.mrk.DiVittori_Bitocchi_2021.gff3.gz
└── G19833.gnm2.mrk.Zitnick_Anderson_Oladzad_2020
    └── phavu.G19833.gnm2.mrk.Zitnick_Anderson_Oladzad_2020.gff3.gz
Vigna/radiata/markers/
├── VC1973A.gnm6.mrk.Sandhu_Singh_2020
│   ├── vigra.VC1973A.gnm6.mrk.Sandhu_Singh_2020.gff3.gz
Vigna/unguiculata/markers/
└── IT97K-499-35.gnm1.mrk.Cowpea1MSelectedSNPs
    └── vigun.IT97K-499-35.gnm1.mrk.Cowpea1MSelectedSNPs.gff3.gz

```

**Comment by sammyjava** [comment]
- [ ] Arachis/hypogaea/markers
- [ ] Glycine/max/markers
- [ ] Phaseolus/vulgaris/markers
- [ ] Vigna/radiata/markers
- [ ] Vigna/unguiculata/markers

**Comment by sammyjava** [comment]
I've done a little renaming, @cann0010  because of how they're called in publications and our naming convention. The one-off Agarwal, Clevenger, et al., 2018 set is now so-named. "Axiom_Arachis2" is used by Massa, et al. and I think others, so I trimmed off the "_SNP" suffix (which I think is a bit silly since the vast majority of our marker sets are SNPs.) To be consistent with that little tweak, I dropped "_SNP" from Axiom_Arachis_58K. I won't scream if you want to put "_SNP" back on, but be sure to make the GFF have the correct name in the "source" field, which wasn't the case before, as well as update genotyping_platform in the /genetic/ files that use them. I did all that and everything is consistent with this update. Most importantly, I created READMEs so I can load these markers.
```
markers
├── Tifrunner.gnm1.mrk.Agarwal_Clevenger_2018
├── Tifrunner.gnm1.mrk.Axiom_Arachis_58K
├── Tifrunner.gnm1.mrk.Axiom_Arachis2
```
